app/assets/stylesheets/application.tailwind.css

```diff
@@ -41,12 +40,12 @@
   }
 
   /* flash notice */
-  .notice { 
-    @apply border border-solid p-5 rounded items-center absolute top-0 h-16 w-fit top-2.5 right-2.5
+  .notice {
+    @apply border border-solid p-5 rounded items-center absolute top-0 h-16 w-fit top-2.5 right-2.5;
   }
 
-  .footer { @apply
-    top-full sticky 
+  .footer {
+    @apply top-full sticky;
```

Comment:
it works even without semicolon, but it's ok

Reply:
That's odd, I want uniformity, I will put it back